src/app/tests/suites/certification/Test_TC_OPCREDS_3_7.yaml

```diff
@@ -11,200 +11,245 @@
       PICS: OPCREDS.S.C0b.Rsp
-      verification: |
-          Verify that AddTrustedRootCertificate command succeeds by sending the status code as SUCCESS
-      disabled: true
-
-    - label:
-          "Step 9: TH1 sends the AddNOC command to DUT with the following
-          fields: NOCValue as Node_Operational_Certificate_TH2 ICACValue as
-          Intermediate_Certificate_TH2 IpkValue as IPK_TH2 CaseAdminSubject as
-          the NodeID of TH2 AdminVendorId as the Vendor ID of TH2"
+      identity: "alpha"
+      command: "AddTrustedRootCertificate"
+      cluster: "Operational Credentials"
+      arguments:
+          values:
+              - name: "RootCACertificate"
+                value: Root_CA_Certificate_TH2
+      # verification: "Verify that AddTrustedRootCertificate command succeeds by sending the status code as SUCCESS"
+    
+    - label: "Step 8: TH1 sends the AddNOC command to DUT with the following fields: NOCValue as Node_Operational_Certificate_TH2. ICACValue as Intermediate_Certificate_TH2. IpkValue as IPK_TH2. CaseAdminSubject as the NodeID of TH2. AdminVendorId as the Vendor ID of TH2."
       PICS: OPCREDS.S.C06.Rsp && OPCREDS.S.C08.Tx
-      verification: |
-          Verify that DUT responds with NOCResponse with status code OK
-      disabled: true
-
-    - label: "Step 10: TH2 starts discovery of DUT using Operational Discovery"
-      verification: |
-
-      disabled: true
-
-    - label:
-          "Step 11: TH2 opens a CASE session with DUT over operational network"
-      verification: |
-          DUT is able to open the CASE session with TH2
-      disabled: true
-
-    - label: "Step 12: TH2 sends CommissioningComplete command"
+      identity: "alpha"
+      command: "AddNOC"
+      cluster: "Operational Credentials"
+      arguments:
+          values:
+              - name: "NOCValue"
+                value: Node_Operational_Certificate_TH2
+              - name: "ICACValue"
+                value: Intermediate_Certificate_TH2
+              - name: "IPKValue"
+                value: IPK_TH2
+              - name: "CaseAdminSubject"
+                value: Commissioner_Node_Id_TH2
+              - name: "AdminVendorId"
+                value: 0xFFF1
+      response:
+          values:
+              - name: "StatusCode"
+                value: 0
+      # verification: "Verify that DUT responds with NOCResponse with status code OK"
+
+    - label: "Step 9: TH2 starts discovery of DUT using Operational Discovery"
+      # PICS: ""
+      # verification: ""
+      # Disabling this step as this occurs from the AddNOC command being run
+      disabled: true
+
+    - label: "Step 10: TH2 opens a CASE session with DUT over operational network"
+      # PICS: ""
+      # verification: "DUT is able to open the CASE session with TH2"
+      # Disabling this step as this occurs from the AddNOC command being run
```

Comment:
AddNOC does not establish CASE on the new fabric...

Does this actually need a "WaitForCommissionee" on the new fabric (for the node id we passed to IssueNocChain) to work right?  I'm _really_ surprised this is working.  I think adding the WaitForCommissionee here, which exactly corresponds to "open a CASE session", would be a good idea.

Reply:
@bzbarsky-apple Added to commit 5bf67b8f8026b5e410e5eca79b9ae9140b8149f7 to ensure the CASE will happen here